src/modules/lv2/factory.c

```diff
@@ -0,0 +1,261 @@
+                int j = desc->status_port_indicies[i];
+                p = mlt_properties_new();
+                snprintf(key, sizeof(key), "%d", mlt_properties_count(params));
+                mlt_properties_set_data(params,
+                                        key,
+                                        p,
+                                        0,
+                                        (mlt_destructor) mlt_properties_close,
+                                        NULL);
+                snprintf(key, sizeof(key), "%d[*]", j);
+                mlt_properties_set(p, "identifier", key);
+                add_port_to_metadata(p, desc, j);
+                mlt_properties_set(p, "readonly", "yes");
+            }
+
+            p = mlt_properties_new();
+            snprintf(key, sizeof(key), "%d", mlt_properties_count(params));
+            mlt_properties_set_data(params, key, p, 0, (mlt_destructor) mlt_properties_close, NULL);
+            mlt_properties_set(p, "identifier", "instances");
+            mlt_properties_set(p, "title", "Instances");
+            mlt_properties_set(p,
+                               "description",
+                               "The number of instances of the plugin that are in use.\n"
+                               "MLT will create the number of plugins that are required "
+                               "to support the number of audio channels.\n"
+                               "Status parameters (readonly) are provided for each instance "
+                               "and are accessed by specifying the instance number after the "
+                               "identifier (starting at zero).\n"
+                               "e.g. 9[0] provides the value of status 9 for the first instance.");
+            mlt_properties_set(p, "type", "integer");
+            mlt_properties_set(p, "readonly", "yes");
+
+            if (type == mlt_service_filter_type) {
+
+                p = mlt_properties_new();
+                snprintf(key, sizeof(key), "%d", mlt_properties_count(params));
+                mlt_properties_set_data(params,
+                                        key,
+                                        p,
+                                        0,
+                                        (mlt_destructor) mlt_properties_close,
+                                        NULL);
+                mlt_properties_set(p, "identifier", "wetness");
+                mlt_properties_set(p, "title", "Wet/Dry");
+                mlt_properties_set(p, "type", "float");
+                mlt_properties_set_double(p, "default", 1);
+                mlt_properties_set_double(p, "minimum", 0);
+                mlt_properties_set_double(p, "maximum", 1);
+                mlt_properties_set(p, "mutable", "yes");
+                mlt_properties_set(p, "animation", "yes");
+            }
+        }
+    }
+#endif
+
+    return result;
+}
+
+MLT_REPOSITORY
+{
+
+#ifdef GPL
```

Comment:
Here we can clearly see there are no MLT services without "GPL." So, if GPLv2 is required, then can be easily enforced in the top-level `CMakeLists.txt` and no need to clutter the code in this module with these conditionals.